specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/computeRPCommon.json

```diff
@@ -1053,6 +1065,10 @@
             "lun"
           ],
           "description": "Specifies the parameters that are used to add a data disk to a virtual machine. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)."
+        },
+        "diskControllerType": {
+          "$ref": "#/definitions/DiskControllerType",
+          "description": "Specifies the disk controller type configured for the VM. <br><br> Minimum api-version: 2022-08-01"
```

Comment:
The comment about the VM update for the diskControllerType was valid why was that removed, pls. add it back.